<?xml version="1.0"?>
<launch>

	<arg name="sensor_name" default="kinect01" />
	
	<!-- Apriltag detector -->
	<node pkg="apriltags_ros" name="$(arg sensor_name)_tag_detector" type="apriltag_detector_node">
		<remap from="image_rect" to="/$(arg sensor_name)/hd/image_color_rect"/>

		<remap from="camera_info" to="/$(arg sensor_name)/hd/camera_info"/>

		<remap from="tag_detections" to="/camera_network/tag_detections"/>
		<remap from="tag_detections_pose" to="/camera_network/tag_detections_pose"/>

		<param name="tag_family" value="36h11"/>

		<rosparam param="tag_descriptions">[
                <!-- Big apriltag -->
                {id: 0, size: 0.570, frame_id: tag_0}]
                <!-- Small apriltag 
                {id: 0, size: 0.1535, frame_id: tag_0}]-->
        </rosparam>
	</node>
  
</launch>

